SuperBuild/External_ITK.cmake

```diff
@@ -33,7 +33,7 @@ if(NOT DEFINED ITK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_TAG
-    "e18566d53a1b8a913cd460e70569e9b485137fba" # slicer-v5.4.0-2024-05-16-311b706
+    "c0202e92b1bf7fb89601cfc5b0991fdd2cd76c1d" # slicer-v5.4.0-2024-05-16-311b706
```

Comment:
Following https://github.com/Slicer/Slicer/pull/8016#issuecomment-2443138236, this will have to be updated.